app/assets/stylesheets/components/toggles.scss

```diff
@@ -0,0 +1,69 @@
+.c-toggle {
+  --gap: 2px;
+  width: 2.75em;
+  height: 1.5em;
+  display: inline-flex;
+  cursor: pointer;
+  position: relative;
+
+  $this: '.c-toggle';
+
+  &__control {
+    width: 100%;
+    height: 100%;
+    display: inline-flex;
+    background: var(--toggle-rail-bg);
+    border-radius: 1000px;
+    transition: var(--transition-props);
+
+    @media (prefers-reduced-motion: reduce) {
+      transition-duration: 0ms;
+    }
```

Comment:
Is this not covered by our `reset.scss` already? (same question with other instance below)

Reply:
Of course it is... I forgot :D

Reply:
fixed.